content/docs/user-guide/data-management/remote-storage/azure-blob-storage.md

```diff
@@ -219,3 +219,14 @@ See `dvc remote modify` for more command usage details.
 
 [Azure documentation]:
   https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob#other-client--per-operation-configuration
+
+### Azure Remote Configuration 
```

Comment:
I don't think we need a separate `###` - it should be just part of the existing list (`url`, timeouts, ect). it is the same: some additional config option.